ui/pages/confirmations/components/simulation-details/balance-change-row.tsx

```diff
@@ -48,7 +67,25 @@ export const BalanceChangeRow: React.FC<{
         style={{ minWidth: 0 }}
       >
         <Box display={Display.Flex} flexDirection={FlexDirection.Row} gap={1}>
-          <AmountPill asset={asset} amount={amount} />
+          {onEdit && (
```

Comment:
This is a great addition, and directly addresses a pain point recently reported internally. 🔥 See https://consensys.slack.com/archives/C03ETQA9EPK/p1739549199067819